### DISCUSSION TITLE
rust and numeric computation

### DISCUSSION THREAD

**Discussion by jimy-byerley** [dimforge/nalgebra#1367] [Ideas]
Hello there
I am a random engineer using nalgebra for numeric computation. 

I just [posted on the rust internals forum](https://internals.rust-lang.org/t/rust-and-numeric-computation/20425) about missing features in rust to allow numeric computation code (and crates) better.
I would be interested in having the opinion of the nalgebra community - especially the nalgebra developper - about it :)

**Comment by sebcrozet** [comment]
Hey, thank you for sharing! Your thoughts are making a lot of sense. I added an answer to that post.

**Reply by jimy-byerley** [reply]
Thanks for this !
